Python 3.6 deprecated (#25768)
Azure/azure-sdk-for-python: sdk/servicebus/azure-servicebus/CHANGELOG.md

### Other Changes

> - Python 3.6 is no longer supported. Please use Python version 3.7 or later.

Reviewer: This message should go under the version line above, and look like:
```suggestion
This version and all future versions will require Python 3.7+. Python 3.6 is no longer supported.
```

Please take a look at version 7.5.0 in this CHANGELOG as an example.
